components/Common/AvatarGroup/index.tsx

```diff
@@ -0,0 +1,62 @@
+'use client';
+import * as Avatar from '@radix-ui/react-avatar';
+import { useState } from 'react';
+import type { FC } from 'react';
+import styles from './index.module.css';
+
+type AvatarGroupProps = {
+  avatars: {
+    src: string;
```

Comment:
Also being very honest, the Avatar component should be agnostic to "GitHub", hence I think the only prop they should receive is a SRC parameter.

At maximum an `alt` optional attribute (both for accessibility) and when the image gives 404 we can render the circle with the text, for example.

The Avatar Component should **not** be GitHub bound. Nor use GitHub usernames as parameters. This is overcomplexing the code and making it limited to GitHub avatars.

Reply:
I will take a look to that